buildscripts/build-managed.sh

```diff
@@ -50,6 +49,55 @@ build_managed_corert()
     fi
 }
 
+get_official_cross_builds()
```

Comment:
It would be nice to add a comment here why it is done this way (that there are no armel tizen nuget packages getting published today). 

This is something that we will likely keep iterating on to get a more sustainable model.